### PR TITLE
Update year Copyright License

### PR DESCRIPTION
**Pull request by roramirez**

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 The MIT License (MIT)
 =====================
 
-Copyright © 2016-2017 Michael Teeuw
+Copyright © 2016-2019 Michael Teeuw
```
Comment (lipis) [inline LICENSE.md]:
```suggestion
Copyright © Michael Teeuw
```

Comment (roramirez) [reply LICENSE.md]: Why?

Comment (lipis) [reply LICENSE.md]: So no more year changes/updates..

Comment (roramirez) [reply LICENSE.md]: Ah, ok. But maybe this to help us to keep in mind about how long old is the project.

Comment (lipis) [reply LICENSE.md]: Not so useful info.. if it makes sense.. add somewhere in the readme when the project started.. but you can always check the history..

Comment (lipis) [reply LICENSE.md]: https://github.com/MichMich/MagicMirror/graphs/contributors

Comment (roramirez) [reply LICENSE.md]: Sound good, but I can see the project year in the copyright is more accord to http://www.gnu.org/licenses/gpl-howto.html

**Pull request merged**